etc/cas.properties

```diff
@@ -38,6 +38,15 @@ cas.osf.login.url=/login?
 cas.logout.url=/logout
 cas.institution.login.url=/login?campaign=institution
 
+##
+# Login Rate Limiting
+#
+cas.rateLimiting.failureRangeInSeconds=1
+cas.rateLimiting.failureThreshold=5
+cas.rateLimiting.usernameParameter=username
+cas.rateLimiting.startDelay=1000
+cas.rateLimiting.repeatInterval=5000
```

Comment:
Config Update: server settings for rate limiting. The rate 5 requests per second is a good balance where normal user experience is not affected and programmatically cracking passwords (brute-force) is prevented.